FutureNorthants/northants-design-system
# Update all dependencies to latest versions

Update all npm dependencies to latest versions.

Jest v27 required setting a [testEnvironment variable](https://testing-library.com/docs/react-testing-library/setup/#jest-27) which I couldn't find in any Jest docs...

The babel updates lead to notices in the console when doing `npm run dev`. It suggested adding these to the babel config to remove the notices. 

## To Test
- Checkout the branch, `npm install && npm run dev` to test the storybook launches and then test pages
- `npm run build` to ensure it builds with rollup
- `yalc publish` then include it in the frontend with `yalc add northants-design-system && yarn install && yarn build && yarn start`. Test the frontend website loads and has no JS errors in the console.
